README-Prototype.md

```diff
@@ -0,0 +1,195 @@
+# esdl Tool
+
+The esdl tool aids with creating and managing ESDL-based and Dynamic ESDL services on an HPCC cluster. It consists of several different commands.
+
+## Overall notes for Tim and Tony
+
+Here is my overview of the design direction:
+
+1. Think of the manifest as config-as-code. The operator using the manifest to generate output shouldn't need to know which options to pass on the command line.
+2. Collapse the template file into the manifest.
+3. Think of the template as having two kinds of markup
+    1. operations- Like `<include>` which will have an associated namespace and an explicit prefix.
+    2. boilerplate- Like `<Method>` which gives structure to the output. These elements will most often use the defaut 'no prefix'.
```

Comment:
"which describes the objects being configured"?

Reply:
OK yeah your wording makes more sense.